setup/monitoring/index.md

```diff
@@ -51,6 +58,93 @@ dashboards](#supplied-dashboards) to your third-party system of choice.
 
 See also [`hal config metric-stores`](/reference/halyard/commands/#hal-config-metric-stores).
 
+### Configuring metric filters
+
+The daemon has a "metric_filter_dir" property that can point to a directory containing
+metric filter specifications. The default directory is /opt/spinnaker-monitoring/filters.
```

Comment:
Recommend mentioning `default.yml` first?

"You can configure filters for all services in `default.yml`, and override the filters per service in `<service>.yml.